util/common/traces/generate.go

```diff
@@ -0,0 +1,43 @@
+package traces
+
+import (
+	"github.com/aws/amazon-cloudwatch-agent-test/util/common/traces/common"
+	"github.com/aws/amazon-cloudwatch-agent-test/util/common/traces/xray"
+	"time"
+)
+
+func StartTraceGeneration(receiver string, agentConfigPath string, agentRuntime time.Duration, traceSendingInterval time.Duration) error {
+	cfg := common.TraceTestConfig{
+		Generator:       nil,
+		Name:            "",
+		AgentConfigPath: agentConfigPath,
+		AgentRuntime:    agentRuntime,
+	}
+	xrayGenCfg := common.TraceGeneratorConfig{
+		Interval: traceSendingInterval,
+		Annotations: map[string]interface{}{
+			"test_type": "simple_otlp",
+		},
+		Metadata: map[string]map[string]interface{}{
+			"default": {
+				"nested": map[string]interface{}{
+					"key": "value",
+				},
+			},
+			"custom_namespace": {
+				"custom_key": "custom_value",
+			},
+		},
+	}
+	switch receiver {
+	case "xray":
+		cfg.Generator = xray.NewLoadGenerator(&xrayGenCfg)
+		cfg.Name = "xray-performance-test"
+	case "otlp":
+		panic("Only supports xray for now.")
```

Comment:
I can't see any good reason to panic. Just return an error instead. 
```suggestion
		return errors.New("OTLP is not supported")
```
Also nitpick - "for now" will become stale as soon as you commit this. Be more concise. The issue isn't that we only support XRay. The issue is we don't support OTLP.

Reply:
If we're not supporting it, then we can just have the "otlp" case fallthrough to the default. There's no reason to call it out.

```suggestion
   default:
		return fmt.Errorf("%s is not supported.", receiver)
```